cookbooks/bcpc-hadoop/templates/default/hv_hive-site.xml.erb

```diff
@@ -131,10 +131,20 @@
 <!-- HiveServer2 Security Configuration -->
 <property>
   <name>hive.server2.authentication</name>
-  <value>KERBEROS</value>
+  <value><%= node[:bach][:hive][:hiveserver2][:authentication][:method] %></value>
+</property>
+<% if node[:bach][:hive][:hiveserver2][:authentication][:method].upcase == "LDAP" %>
```

Comment:
Nit: There isn't going to be blank line injected, in the resulting xml, in place of this ruby code block. Which means the properties before and after this line won't have a separating blank line as the rest of the properties.